charts/opentelemetry-collector/values.yaml

```diff
@@ -73,7 +80,6 @@ imagePullSecrets: []
 # OpenTelemetry Collector executable
 command:
   name: otelcol
-  # name: otelcontribcol
```

Comment:
Can we bring back the comments? they seem to be useful

Reply:
If we want comments let's make sure they are actual readable sentences and don't look like a commented code that we forgot to delete. 

A comment like the one that was on line 76 is not very helpful, it is confusing. I don't know if it is supposed to be uncommented or deleted. How do I know it should replace the previous line?

Reply:
@dmitryax how do you want to move forward? Address this in this PR or a separate PR?

Reply:
Adding a comment would be helpful. I'm good with either way: addressing it here or in separate PR.

Reply:
@astencel-sumo up to you. I can merge now or merge for you to address it.

Reply:
I'll add a comment to this.

Reply:
Added comment in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/36/commits/04c8f4734cf4a5a48fbab40bc1ed8b6ba85da618.